CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+## 0.1.0 (2021-12-16)
+
+- Initial release of the WASM Miden playground
+- Adding six examples for written in Miden assembly
+- Adding a simple front to show the Miden assembly code and execute it
```

Comment:
1. I think there's either an extra word (for) or a missing word in the second point
2. I would say "front-end" instead of "front"
3. nit: I would probably use past tense (e.g. "Added ..."), since that's what we do in the Miden VM and Winterfell changelogs

Reply:
haha thanks a lot and sorry, I could have read it again before submitting...